packages/mobile/src/components/core/HarmonySelectablePill.tsx

```diff
@@ -90,18 +92,20 @@ export const HarmonySelectablePill = (props: HarmonySelectablePillProps) => {
 
   const handlePressIn = useCallback(
     (event: GestureResponderEvent) => {
+      if (isDisabled) return
       onPressIn?.(event)
       handlePressInScale()
     },
-    [handlePressInScale, onPressIn]
+    [handlePressInScale, isDisabled, onPressIn]
   )
 
   const handlePressOut = useCallback(
     (event: GestureResponderEvent) => {
+      if (isDisabled) return
```

Comment:
Huh. So `Pressable` still fires the callbacks even when you pass it a true `disabled` prop?
If that's true, I'm wondering if it's cleaner to not change these callbacks based on `disabled`, but just not pass them to the Pressable in that case?

```
<Pressable onPressIn={ disabled ? undefined : handlePressIn } />
```
Maybe just personal preference, but I do know that some components have optimized inner logic based on presence / absence of a callback (we do this in some of our own components)

Reply:
oh thanks, i'll try this out!